test/functional/page_objects/common_page.ts

```diff
@@ -67,17 +67,17 @@ export function CommonPageProvider({ getService, getPageObjects }: FtrProviderCo
      * @param appUrl Kibana URL
      */
     private async loginIfPrompted(appUrl: string, insertTimestamp: boolean) {
+      // Disable the welcome screen. This is relevant for environments
+      // which don't allow to use the yml setting, e.g. cloud production.
+      // It is done here so it applies to logins but also to a login re-use.
+      await browser.setLocalStorageItem('home:welcome:show', 'false');
```

Comment:
Is it a problem only for the case of login? Maybe it is better to set it up after driver instance is created?

Reply:
I was thinking if the Kibana home page can have some attribute stating it is running on cloud..

Reply:
My first idea was to put this logic into [beforeTestSuite](https://github.com/elastic/kibana/blob/master/test/functional/services/remote/remote.ts#L115), which would be similar to a setup after driver instance creation. But at that stage the browser storage can't be set:
```
Storage is disabled inside 'data:' URLs.
```
So we basically need some actual URL loaded before we can set this, which makes it harder to find a good place for this logic as the entry points can be very different.
The login page seemed to be a good thing as cloud testing always involves a login. But 
a) we have two different page objects handling the login page. This could be handled by adding the logic to both of them.
b) tests that start with the navigation to an app will re-use an existing login and don't involve the login page. But the browser storage is still cleared between tests, so the welcome screen is enabled again.

Not sure if an attribute to identify cloud usage on the welcome page solves the issues, since we still need to handle it and there are a lot of pathes that could lead to a displayed welcome page. IMO having something like `closeWelcomePageIfNeeded` and calling it from all relevant places would be a better solution than trying to disable the page via the browser storage. But `calling it from all relevant places` is the tricky part here.

Reply:
ok, let's keep it this way for now